pkgs/misc/pylode/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonApplication
+, dateutil
+, falcon
+, fetchFromGitHub
+, gunicorn
+, isodate
+, jinja2
+, markdown
+, rdflib
+, rdflib-jsonld
+, requests
+, six
+, beautifulsoup4
+}:
+
+buildPythonApplication rec {
+  pname = "pyLODE";
+  version = "2.8.6";
+
+  src = fetchFromGitHub {
+    owner = "RDFLib";
+    repo = pname;
+    rev = version;
+    sha256 = "0zbk5lj9vlg32rmvw1himlw63kxd7sim7nzglrjs5zm6vpi4x5ch";
+  };
+
+  propagatedBuildInputs = [
+    dateutil
+    falcon
+    gunicorn
+    isodate
+    jinja2
+    markdown
+    rdflib
+    rdflib-jsonld
+    requests
+    six
+    beautifulsoup4
+  ];
+
+  meta = with lib; {
+    description = "An OWL ontology documentation tool using Python and templating, based on LODE";
+    homepage = "https://github.com/RDFLib/pyLODE";
```

Best practice now is to add a ``meta.changelog`` attribute if it exists.

```suggestion
    homepage = "https://github.com/RDFLib/pyLODE";
    changelog = "https://github.com/RDFlib/pyLODE...";
```
Or a more user-friendly documentation site if it exists, e.g. readthedocs

---

The package has nothing of the sort. The only existing documentation is the README of the Github repository.